includes/class-micropub-authorize.php

```diff
@@ -25,6 +25,9 @@ class Micropub_Authorize {
 	// Array of Scopes
 	protected static $scopes = array();
 
+	// Error
+	protected static $error = null;
```

Comment:
this needs to get set to NULL somewhere at the beginning of request handling, right? otherwise, since it's static, when it gets set to an error in one request, it will keep that value across other future requests.

Reply:
...more importantly, using long-lived state like this to affect control flow across multiple different parts of the code is hard to follow and error prone. we currently use eg `static::error()` to immediately stop and return an error at different points right now. i assume that no longer works because it's not the REST API way? can we find anything similar for REST? i'd much rather do that instead.

Reply:
As noted, and will document, this is only for authorization. Otherwise, the default return message doesn't allow us to explain why it was rejecting, it will just say unauthorized. All Micropub errors are returned directly to the user.